HISTORY.rst

```diff
@@ -3,6 +3,18 @@
 History
 -------
 
+4.0.0
+++++++++++++++++++
+
+* IMPORTANT: Python 2.7 and 3.5 support has been drop. Python 3.6 or greater
```

Comment:
"dropped"?

Reply:
Doh, I was so focused on whether the plural or singular was actually right that I totally missed that.

Reply:
I cleaned up the history a bit and applied the changes to the original commit that added the line.